Source/player.cpp

```diff
@@ -3600,12 +3388,8 @@ void ModifyPlrMag(Player &player, int l)
 	player._pMagic += l;
 	player._pBaseMag += l;
 
-	int ms = l << 6;
-	if (player._pClass == HeroClass::Sorcerer) {
-		ms *= 2;
-	} else if (player._pClass == HeroClass::Bard) {
-		ms += ms / 2;
-	}
+	int ms = l;
+	ms *= PlayersData[static_cast<size_t>(player._pClass)].chrMana;

```

Comment:
I don't think there is a reason to multiply?
```suggestion
	int ms = PlayersData[static_cast<size_t>(player._pClass)].chrMana;
```